webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/TemplatesCard/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@patternfly/react-core';
+import { PencilAltIcon } from '@patternfly/react-icons';
+import { TableComposable, Tr, Tbody, Td } from '@patternfly/react-table';
+import { translate as __ } from '../../../../../../common/I18n';
+import { foremanUrl } from '../../../../../../common/helpers';
+import { useAPI } from '../../../../../../common/hooks/API/APIHooks';
+import SkeletonLoader from '../../../../../common/SkeletonLoader';
+import DefaultLoaderEmptyState from '../../../../DetailsCard/DefaultLoaderEmptyState';
+import CardTemplate from '../../../../Templates/CardItem/CardTemplate';
+import { STATUS } from '../../../../../../constants';
+
+const TemplatesCard = ({
+  hostName,
+  hostDetails: { managed },
+  isExpandedGlobal,
+}) => {
+  const templatesUrl = foremanUrl(`/api/hosts/${hostName}/templates`);
+  const TemplateTypeTitle = __('Template Type');
+  const {
+    response: {
+      templates,
+      edit_provisioning_templates: editTemplatePermission,
+    },
+    status,
+  } = useAPI('get', templatesUrl);
+  const editTemplateUrl = id => `/templates/provisioning_templates/${id}/edit`;
+  if (!managed || !templates?.length) return null;
```

Comment:
I registered a RHEL8 host but I couldn't see the TemplatesCard because apparently my host is not "managed." Is this the correct behavior?

Reply:
https://github.com/theforeman/foreman/blob/develop/app/views/hosts/show.html.erb#L34
Thats what happens in the old form so I copied it

Reply:
I _think_ it's correct because I assume "managed" means "I provisioned this with Foreman so it should have templates." but just wanted to double check. @ares ?

Reply:
You're right, the "managed" meaning is really that the host is supposed to be provisioned from the Foreman. There's just one exception - host init and registration templates. They can be assigned even for unmanaged hosts. For that reason, if it does not cause any issues, I'd consider showing the templates for both host types.